cfg/dancer.conf

```diff
@@ -1,10 +1,18 @@
-PerlPassEnv MYSQL_USER
-PerlPassEnv MYSQL_PASSWORD
-PerlPassEnv MYSQL_DATABASE
-PerlPassEnv MYSQL_ROOT_PASSWORD
-
 <Location />
   SetHandler perl-script
   PerlResponseHandler Plack::Handler::Apache2
   PerlSetVar psgi_app /opt/openshift/src/bin/app.psgi
 </Location>
+
+
+<Perl>
+local $ENV{PATH} = '/usr/bin';
+local $_;
+
+for (`env`) {
+    $_=~s/(.*?)=.*/$1/;
+    push @PassEnv, $_;
+}
+</Perl>
```

Comment:
@ewolinetz check out what i learned to do tonight.....  you probably want to update your sti-perl PR to do it this way.
@brenton may also be of interest to you.  only works if mod_perl is enabled.